add docs for test id attribute (#379)
OctoMind-dev/mintlify-docs: runtime/testid-attribute.mdx

---
title: Test Id Attribute
description: "Configure the html attribute name you use for test automation"
icon: "code"
---

Our Agent looks at specific parts of your HTML.
One of the attributes it pays special attention to is the test id.
Different frameworks use different values (playwright for example uses [data-testid](https://playwright.dev/docs/locators#locate-by-test-id)) or you might be using a custom attribute for other tests you have.

> To help our agent understand your page better you can tell it how this attribute is named in your case in the settings.

Reviewer: ```suggestion
To help our agent understand your page better, you can configure this attribute name in your project settings.
```
